src/lib/config/config.ts

```diff
@@ -18,6 +18,8 @@ type Config = {
     scrollingLazy: boolean;
     scrollingCentered: boolean;
     scrollingGrouped: boolean;
+    touchpadGestures: boolean;
+    naturalScrolling: boolean;
```

Comment:
```suggestion
    gestureScroll: boolean;
    gestureScrollInvert: boolean;
```

I think these names will work best with potential other gestures in the future. And it makes it clear that these settings are related.